src/Morphic-Widgets-Windows/SystemWindow.class.st

```diff
@@ -1896,6 +1896,33 @@ SystemWindow >> taskbarMoveRight [
 	self moveTaskbarTaskToOffset: 1
 ]
 
+{ #category : 'move' }
+SystemWindow >> taskbarMoveTo [
+	"Move the current task button to the prompted location"
+	
+	| taskBarSize taskLocation desiredLocation answer |
+	
+	taskBarSize := self worldTaskbar orderedTasks size.
+	taskLocation := self worldTaskbar orderedTasks indexOf: self taskbarTask.
+
+	answer := UIManager default
```

Comment:
I know that Stephane wants us to reduce the references to UIManager. But Spec is loaded after SystemWindow so I am not so sure of how to do what you are doing without UIManager. Maybe @Ducasse or @estebanlm know?

Reply:
Just needs a simple way to ask for input - so presumably there is a recommended way to do this going forward and I can adjust and resubmit.

Worth adding, I've tried to use the taskbar primitives already their so keep the api as small as possible in case there are changes proposed in this area.

Reply:
:( shitty architecture strikes back. Now SystemWindow is in morphic so I do not care. we should kill it alltogether.
Now this behavior will have to be redone on top of Toplo (and not the spec backend)